src/contentscript/core.ts

```diff
@@ -718,4 +719,9 @@ export default class Core {
   public state<T>(defaultState: T, type?: string) {
     return new State<T>(defaultState, type)
   }
+
+  public connectedAccounts() {
+    const connectedAccounts = new ConnectedAccounts()
```

Comment:
The `ConnectedAccounts` class is instantiated every time here. You can move it to the Core class properties and reuse:

```typescript
class Core {
   private _connectedAccounts = new ConnectedAccounts()
   
   public connectedAccounts() {
       return this._connectedAccounts
   }
}
```

or just


```typescript
class Core {
   public connectedAccounts = new ConnectedAccounts()
}
```

But it affects Dapplets Core API Core.connectedAccounts() => Core.connectedAccounts

Reply:
`public connectedAccounts = new ConnectedAccounts()`